src/server/visual-diff-reporter.js

```diff
@@ -112,7 +112,7 @@ export function visualDiffReporter({ reportResults = true } = {}) {
 			cpSync(inputDir, tempDir, { force: true, recursive: true });
 			writeFileSync(join(tempDir, 'data.js'), `export default ${json};`);
 
-			execSync(`rollup -c ${join(__dirname, './rollup.config.js')}`);
+			execSync(`npx rollup -c ${join(__dirname, './rollup.config.js')}`, { stdio: 'pipe' });
```

Comment:
`npx` fixes `command not found` error, which I believe is necessary now the way this runs.

`stdio` fixes the double output, though I'm not sure why, since it's supposed to be the default. May need to make this `ignore` if it spits out other bad things.